storage/boltdb/net-state.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package boltdb
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/boltdb/bolt"
+)
+
+// File Path and Value are saved to boltdb
+type File struct {
+	Path  string `json:"path"`
+	Value string `json:"value"`
+}
+
+const (
+	fileBucketName = "files"
+)
+
+var (
+	errFileNotFound = Error.New("error file not found")
+)
+
+func (client *Client) Put(file File) error {
+	return client.db.Update(func(tx *bolt.Tx) error {
+		b, err := tx.CreateBucketIfNotExists([]byte(fileBucketName))
+		if err != nil {
+			return err
+		}
+
+		fileKey := []byte(file.Path)
+
+		fileBytes, err := json.Marshal(file.Value)
```

Comment:
file.Value is already a string, so you could just cast it to a []byte array.

```
fileBytes := []byte(file.Value)
```

in fact, you could do this:

```
type File struct {
  Path string
  Value []byte
}
```

and then `file.Value` is already `[]byte`